Modules/UserManager/Controllers/UserManageController.cs

```diff
@@ -17,47 +17,60 @@ public class UserManageController : DnnController
         [ModuleAction(ControlKey = "Edit", TitleKey = "AddItem")]
         public ActionResult Index(double? take, int? pageIndex, string filter, int? goToPage, string search, string orderBy, string order)
         {
-            var takeValue = take ?? default;
-            var pageIndexValue = take == null ? default : pageIndex.Value;
-            var portalId = ModuleContext.PortalId;
-            var portalSettings = ModuleContext.PortalSettings;
-            string serverUrl = $"{Request.Url.Scheme}://{portalSettings.PortalAlias.HTTPAlias}";
-            switch (filter)
+            // Check if the user is authenticated
+            bool isAuthenticated = Request.IsAuthenticated;
+
+            if (!isAuthenticated)
             {
-                case "8":
-                    ViewBag.Filter = "All";
-                    break;
-                case "0":
-                    ViewBag.Filter = "Authorized";
-                    break;
-                case "1":
-                    ViewBag.Filter = "Unauthorized";
-                    break;
-                case "2":
-                    ViewBag.Filter = "Deleted";
-                    break;
-                case "3":
-                    ViewBag.Filter = "SuperUsers";
-                    break;
-                default:
-                    filter = "0";
-                    ViewBag.Filter = "Authorized";
-                    break;
+                // User not authenticated, return error message
+                string errorMessage = "You do not have the necessary security permissions to use this application.";
```

Comment:
@alejoroman0605 This needs to be localized, and NOT hard-coded in English.

Reply:
@WillStrohl I uploaded an update to resolve the Localization issue. I changed all the static texts to use the localization from the localization files. This also resolves the issue https://github.com/UpendoVentures/Upendo-Dnn-UserManager/issues/31. I uploaded it all in this PR because the two issues were strongly related, and what I did was I fixed the bugs using Localization in the whole module.